dashboard/app/controllers/learning_goal_evaluations_controller.rb

```diff
@@ -0,0 +1,39 @@
+class LearningGoalEvaluationsController < ApplicationController
+  authorize_resource
+
+  def create
+    learning_goal_evaluation = LearningGoalEvaluation.new(learning_goal_evaluation_params)
+    learning_goal_evaluation.teacher_id = current_user.id
+
+    learning_goal_evaluation.save!
+
+    render json: learning_goal_evaluation
+  end
+
+  def update
+    params.require(:id)
+    learning_goal_evaluation = LearningGoalEvaluation.find_by_id(params[:id])
+
+    if learning_goal_evaluation &&
+      learning_goal_evaluation.teacher_id == current_user.id &&
+      learning_goal_evaluation.update(learning_goal_evaluation_params)
+      render json: learning_goal_evaluation
+    else
+      render status: :not_found
+    end
```

Comment:
Two things:
1) Not finding the row (or wanting the user to think we couldn't) is a fundamentally different thing from being unable to update the row. I suggest using different error codes so the client can handle these errors differently.
2) The `render` method expects a response body. `head` is for when you want to return an HTTP status code without any additional body content.

```suggestion
    if learning_goal_evaluation&.teacher_id == current_user.id
      if learning_goal_evaluation.update(learning_goal_evaluation_params)
        render json: learning_goal_evaluation
      else
        head :bad_request
      end
    else
      head :not_found
    end
```